src/rich_text_editor/model/RichTextEditor.js

```diff
@@ -156,11 +195,13 @@ export default class RichTextEditor {
    */
   syncActions() {
     this.getActions().forEach(action => {
-      const event = action.event || 'click';
-      action.btn[`on${event}`] = e => {
-        action.result(this, action);
-        this.updateActiveActions();
-      };
+      if (!action.state || (action.state && action.state() >= 0)) {
```

Comment:
I think you should call the `state` function with the proper signature `state(rte, doc)`